.github/workflows/ci-cd.yml

```
@@ -21,7 +21,7 @@ jobs:
       - name: Install Node
         uses: actions/setup-node@v4
         with:
-          node-version: 18.x
+          node-version: 20.19.x
```

why not
```suggestion
          node-version: 20.x
```
so it could be get latest v20 shuld there be any fixes/new versions available

---

Same as above.